README.md

```diff
@@ -61,6 +64,12 @@ Instead of overriding `save` and `__init__` in a clunky way that hurts readabili
 
 # Changelog
 
+## Pending
+
+- Drop support for Python < 3.8.
+- Drop support for Django < 3.2.
```

Comment:
Does it makes sense to remove support for those versions? If we keep `django-capture-on-commit-callbacks==1.10.0` we could still support Django 2.x

Reply:
I prefer to drop versions that are EOL. Uesrs on old versions can stick to the old versions of the package.